RecipesManagerWebClient.Web/Network/AuthenticationService.cs

```diff
@@ -25,39 +25,17 @@ public async Task<string> GetAuthTokenAsync()
     {
         var jwtToken = _httpContext.Request.Cookies["accessToken"];
 
-        if (string.IsNullOrEmpty(jwtToken) || IsJwtTokenExpired(jwtToken)) {
-            var webId = _httpContext.Request.Cookies["webId"];
-            if (string.IsNullOrEmpty(webId))
-            {
-                webId = Guid.NewGuid().ToString();
-                _httpContext.Response.Cookies.Append("webId", webId, new CookieOptions { Expires = DateTime.UtcNow.AddDays(180) });
-            }
-
-            var request = new GraphQLRequest
-            {
-                Query = @"
-                    mutation AccessWebGuest($user: AccessWebGuestModelInput!) {
-                        accessWebGuest(model: $user) {
-                            accessToken
-                            refreshToken
-                        }
-                    }",
-                Variables = new { user = new { webId = webId } } 
-            };
-
-            var response = await _graphQLClient.SendMutationAsync<dynamic>(request);
-            var jsonResponse = JsonConvert.SerializeObject(response.Data.accessWebGuest);
-            var tokens = JsonConvert.DeserializeObject<TokensModel>(jsonResponse);
-            _httpContext.Response.Cookies.Append("accessToken", tokens.AccessToken, new CookieOptions { Expires = DateTime.UtcNow.AddDays(180) });
-            _httpContext.Response.Cookies.Append("refreshToken", tokens.RefreshToken, new CookieOptions { Expires = DateTime.UtcNow.AddDays(180) });
-
-            jwtToken = tokens.AccessToken;
+        if (string.IsNullOrEmpty(jwtToken)) {
+            jwtToken = await AccessWebGuest();
+        }
+        if (IsJwtTokenExpired(jwtToken))
+        {
+            jwtToken = await RefreshToken();
```

Comment:
What if refresh was unsuccessful? (Due to refresh token expiry date). Add try catch and redirect to login page